patches/0001-use-released-aws-nitro-enclaves-cose-version.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/Cargo.toml b/Cargo.toml
+index 5c239db2..a05cbe01 100644
+--- a/Cargo.toml
++++ b/Cargo.toml
+@@ -38,6 +38,3 @@ default-members = [
+ ]
+ 
+ resolver = "2"
+-
+-[patch.crates-io]
+-aws-nitro-enclaves-cose = { git = "https://github.com/awslabs/aws-nitro-enclaves-cose/", rev = "6064f826d551a9db0bd42e9cf928feaf272e8d17" }
```

Comment:
just a silly question, if we don't need to patch this going forward, why do we still carry the patch? (it may be obvious but it's monday...)

Reply:
The patch that solves the problem with multi-thread environments is not included in any `aws-nitro-enclaves cose` release yet. Therefore, we need to patch the latest version in the `Cargo.toml` file at workspace level and use a revision that includes the patch.
On the other hand, the downstream crate in Fedora (v0.5.2) is already patched and includes the mentioned fix so we need to remove the workspace modifications to use the official (and patched) v0.5.2 version in Fedora.

Reply:
I've asked to get a proper tag https://github.com/awslabs/aws-nitro-enclaves-cose/pull/115#issuecomment-2690536258 that's probably the best thing and what we've done in the past (far past lol)

Reply:
@runcom already asked in https://github.com/awslabs/aws-nitro-enclaves-cose/issues/114 three weeks ago, no response so far.

Reply:
ah neat, didn't see that - uhm